src/Pretzel.Logic/Templating/Jekyll/LiquidEngine.cs

```diff
@@ -85,13 +85,13 @@ private Hash CreatePageData(PageContext pageContext)
             return x;
         }
 
-        protected override string RenderTemplate(string templateContents, PageContext pageData)
+        protected override string RenderTemplate(string content, PageContext pageData)
```

Comment:
Is this renaming part of a warning fix?

Reply:
warning fix: base class uses `protected abstract string RenderTemplate(string content, PageContext pageData);` hence inherited classes should use the same name.